app/libs/articles_widget.py

```diff
@@ -1,104 +1,39 @@
-import webbrowser
-from PySide2 import QtWidgets, QtGui
+from PySide2 import QtWidgets, QtCore
 from libs.thread_pool import ThreadPool
 from libs.articles.get_articles import Articles
+from libs.widgets.article_itemwidget import ArticlesWidgetItem
 
 
-class ArticlesWidget(QtWidgets.QWidget):
+class ArticlesWidget(QtWidgets.QListWidget):
     def __init__(self, parent=None, ticker=None):
         super(ArticlesWidget, self).__init__(parent)
 
         self.thread_pool = ThreadPool()
         self.get_articles(ticker)
 
+    @QtCore.Slot(str)
     def get_articles(self, ticker):
```

Comment:
Try to keep a homogeneity. All method signals are like this: `_on_event()`.

This wiil be better : 
```
@Qtcore.Slot(str)
def _on_(specific event):
    get_articles(ticker)
```